Specs/DateTime/Dutch/DateExtractor.json

```diff
@@ -2585,5 +2585,77 @@
     "Comment": "1/4 shouldn't recognized as date here",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": []
+  },
+  {
+    "Input": "Ik ga terug op Sep-23-2020.",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "Sep-23-2020",
+        "Type": "date",
+        "Start": 15,
+        "Length": 11
+      }
+    ]
+  },
+  {
+    "Input": "Ik ga terug op September 2020 23.",
```

Comment:
I don't think we need to support space-separated cases like these... It will probably lead to false positive merges if there are number mentions around the date. Let's support only dates with proper separators.
Also, "2020/September/23", with the full month name, should be a supported format. Are there spec cases for it?

Reply:
agreed, removed patterns with space separators. 
All patterns support both cases, full month name and abbreviation (since these are defined in the same regex MonthRegex).
For this particular pattern, I put in the spec the case with abbreviation "2020/Sep/23", but there are other cases that show support for the full month name (e.g. "September-2020-23").

Reply:
To chime in, I haven't seen this case in Dutch at all. `23 september 2020` would make sense, or in a notation like

23/sept/2020, 2020/sept/23 or something similar.

Reply:
I have not added this kind of pattern (23 september 2020) to the specs because it was already covered and the Dutch specs contain it already (for example the sixth case in DateExtractor "Ik ga terug op 12 januari 2016"). The other two cases have been added in the current PR ("Ik ga terug op 23/Sep/2020" and "Ik ga terug op 2020-Sep-23")